### ISSUE TITLE
Tab count clashing with URL in private tabs

### ISSUE THREAD

**Issue by mfsoftworks** [brave/brave-browser#7835]
## Description 
Currently in private tabs, the tab count is displayed over the URL with a transparent background so any long URL is shown with the tab count on top making readability difficult 

## Steps to reproduce 

   1. Open private tabs 
   2. Go to any sufficiently long URL to fill the full URL bar

## Actual result 
Tab count is displayed over URL with the URL underneath making legibility low

## Expected result
Tab count is displayed separately from the URL to be easy to read 

## Issue reproduces how often 
Every time a long URL is opened in private tabs

## Issue happens on 
   - Current Play Store version

## Device details
   - Install type (ARM, x86): ARM
   - Device (Phone, Tablet, Phablet): Phone
   - Android version: 9
   
## Brave version
1.7.0

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by mfsoftworks** [comment]
> This is only an issue when bottom toolbar is disabled. This needs to be checked on beta as well.

You're right, I have bottom bar disabled to increase view space, with bottom bar it's fine

**Comment by srirambv** [comment]
This issue will most probably be fixed on beta.

**Comment by srirambv** [comment]
This needs to be fixed on core as well. Here's a screenshot on the latest nightly build
![image](https://user-images.githubusercontent.com/17010094/72936566-7264ef80-3d35-11ea-89e2-587019916324.png)

We need to move the tab count outside the URL similar to how it looks on normal window.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
